unbxdAutosuggest.js

```diff
@@ -1481,32 +1497,32 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
-				+ '{{#each data.IN_FIELD}}'
-				+ '{{#unbxdIf type "keyword"}}'
-				+ '{{else}}'
-				+ '<li data-index="{{@index}}" data-type="{{type}}" data-value="{{autosuggest}}" data-filtername="{{filtername}}" data-filtervalue="{{filtervalue}}"  data-source="{{source}}">'
-				+ (this.options.inFields.tpl ? this.options.inFields.tpl : this.default_options.inFields.tpl)
-				+ '</li>'
-				+ '{{/unbxdIf}}'
-				+ '{{/each}}'
-				+ '{{/if}}';
-			} else { 
+					+ (this.options.inFields.header ? '<li class="unbxd-as-header">' + this.options.inFields.header + '</li>' : '')
+					+ '{{#each data.IN_FIELD}}'
+					+ '{{#unbxdIf type "keyword"}}'
+					+ '{{else}}'
+					+ '<li data-index="{{@index}}" data-type="{{type}}" data-value="{{autosuggest}}" data-filtername="{{filtername}}" data-filtervalue="{{filtervalue}}"  data-source="{{source}}">'
+					+ (this.options.inFields.tpl ? this.options.inFields.tpl : this.default_options.inFields.tpl)
+					+ '</li>'
+					+ '{{/unbxdIf}}'
+					+ '{{/each}}'
+					+ '{{/if}}';
+			} else {
 				return '{{#if data.IN_FIELD}}'
-				+ (this.options.inFields.header ? '<li class="unbxd-as-header">' + this.options.inFields.header + '</li>' : '')
-				+ '{{#each data.IN_FIELD}}'
-				+ '{{#unbxdIf type "keyword"}}'
-				+ '<li class="unbxd-as-keysuggestion" data-index="{{@index}}" data-value="{{autosuggest}}" data-type="IN_FIELD" data-source="{{source}}">'
-				+ (this.options.inFields.tpl ? this.options.inFields.tpl : this.default_options.inFields.tpl)
-				+ '</li>'
-				+ '{{else}}'
-				+ '<li class="unbxd-as-insuggestion" style="color:' + this.options.theme + ';" data-index="{{@index}}" data-type="{{type}}" data-value="{{autosuggest}}" data-filtername="{{filtername}}" data-filtervalue="{{filtervalue}}"  data-source="{{source}}">'
-				+ 'in ' + (this.options.inFields.tpl ? this.options.inFields.tpl : this.default_options.inFields.tpl)
-				+ '</li>'
-				+ '{{/unbxdIf}}'
-				+ '{{/each}}'
-				+ '{{/if}}';
-			  }
+					+ (this.options.inFields.header ? '<li class="unbxd-as-header">' + this.options.inFields.header + '</li>' : '')
+					+ '{{#each data.IN_FIELD}}'
+					+ '{{#unbxdIf type "keyword"}}'
+					+ '<li class="unbxd-as-keysuggestion" data-index="{{@index}}" data-value="{{autosuggest}}" data-type="IN_FIELD" data-source="{{source}}">'
+					+ (this.options.inFields.tpl ? this.options.inFields.tpl : this.default_options.inFields.tpl)
+					+ '</li>'
+					+ '{{else}}'
+					+ '<li class="unbxd-as-insuggestion" style="color:' + this.options.theme + ';" data-index="{{@index}}" data-type="{{type}}" data-value="{{autosuggest}}" data-filtername="{{filtername}}" data-filtervalue="{{filtervalue}}"  data-source="{{source}}">'
+					+ 'in ' + (this.options.inFields.tpl ? this.options.inFields.tpl : this.default_options.inFields.tpl)
+					+ '</li>'
+					+ '{{/unbxdIf}}'
+					+ '{{/each}}'
+					+ '{{/if}}';
+			}
```

Comment:
Can you correct the indentation?

Reply:
@anantheshadiga Indentation is correct, but now part of second commit
@ganeshparsads  Indentation is correct, but now part of second commit